src/plugins/intel_cpu/src/graph_optimizer.cpp

```diff
@@ -2162,36 +2157,6 @@ void GraphOptimizer::DropDoubleReorders(Graph &graph) {
     }
 }
 
-void GraphOptimizer::FuseBroadcastAndEltwise(Graph &graph) {
-    auto& graphNodes = graph.GetNodes();
-
-    for (auto &graphNode : graphNodes) {
-        if (graphNode->getType() != Type::Generic
-                || graphNode->getTypeStr() != "Broadcast"
-                || graphNode->getChildEdges().size() != 1lu
-                || graphNode->getChildEdgeAt(0)->getChild()->getType() != Type::Eltwise)
-            continue;
-
-        NodePtr& broadcastNode = graphNode;
-        NodePtr eltwiseNode = broadcastNode->getChildEdgeAt(0)->getChild();
-        eltwiseNode->inputShapes[broadcastNode->getChildEdgeAt(0)->getOutputNum()]
-                = broadcastNode->getInputShapeAtPort(0);
-
-        auto& edges = graph.GetEdges();
-        for (size_t i = 1lu; i < broadcastNode->getParentEdges().size(); i++) {
-            auto constParent = broadcastNode->getParentEdgesAtPort(i)[0]->getParent();
-            for (auto it = edges.begin(); it != edges.end(); it++) {
-                if ((*it) == constParent->getChildEdgeAt(0)) {
-                    edges.erase(it);
-                    constParent->remove();
-                    break;
-                }
-            }
-        }
-        graph.DropNode(broadcastNode);
```

Comment:
why have you removed it?

Reply:
After `Generic` is removed, `graphNode->getType() != Type::Generic` will always be true:
https://github.com/openvinotoolkit/openvino/blob/1d03b0813bc66c9231cb908820634ac18e6c31df/src/plugins/intel_cpu/src/graph_optimizer.cpp#L2168-L2173

it will goto `continue ` every time  and not go to line 2175 any more. Then all logic will not be done, we can remove this optimizer function completely.
I guess that this optimizer function was used for pattern of `generic op(Broadcast) -> eltwise op`, and seems that current `eltwise `can support `Broadcast` functionality, we don't need such fuse any more, right? @ilya-lavrenov